pointnet2/train/train_sem_seg.py

```diff
@@ -103,6 +104,10 @@
 
     model = Pointnet(num_classes=13, input_channels=6, use_xyz=True)
     model.cuda()
+    print(model)
+    model_parameters = filter(lambda p: p.requires_grad, model.parameters())
+    params = sum([np.prod(p.size()) for p in model_parameters])
+    print("Model size = %i" % params)
```

Comment:
This can be done as a nice one liner: `sum(param.numel() for param in model.parameters() if param.requires_grad)`